Project.toml

```diff
@@ -41,7 +41,6 @@ WriteVTK = "64499a7a-5c06-52f2-abe2-ccb03c286192"
 
 [compat]
 ArgParse = "1.1"
-CLIMAParameters = "0.1"
```

Comment:
🤔

Reply:
It's temporary to make the docs build now. Because this PR depends on Parameters PR 16. (https://github.com/CliMA/CLIMAParameters.jl/pull/16) 

What I did was in package manager
`rm CLIMAParameterds`
`add CLIMAParameters#aj/more_micro`

Once the PR to Parameters is accepted I'll revert that